pkg/cmd/server/bootstrappolicy/infra_sa_policy.go

```diff
@@ -66,6 +66,9 @@ const (
 	// TemplateInstance backend for most of the heavy lifting.
 	InfraTemplateServiceBrokerServiceAccountName = "template-service-broker"
 	TemplateServiceBrokerControllerRoleName      = "system:openshift:template-service-broker"
+
+	InfraRegistryManagementControllerServiceAccountName = "registry-management-controller"
```

Comment:
this will make my life in https://github.com/openshift/origin/pull/14317 harder :-) is it possible to base this on top of the refactoring?